anthos-bm-openstack-terraform/README.md

```diff
@@ -1,13 +1,53 @@
-# Terraform example to create OpenStack VMs for Anthos
-This repository contains an example of what OpenStack resources you need to
-create for setting up Anthos on Bare Metal. The following resources are
-created as part of the Terraform module:
-- External Load Balancer using Octavia LBaaS
+## Anthos on Bare Metal on OpenStack with Terraform
```

Comment:
View [ markdown rendered file](https://github.com/GoogleCloudPlatform/anthos-samples/blob/0740ce56e50d72303e437a7d90b8d2f96e3af4ab/anthos-bm-openstack-terraform/README.md)

Reply:
Yes, there will be a `devsite` doc accompanying this: [go/abm-on-openstack-devsite](https://go/abm-on-openstack-devsite). It's owned by @grca and reviewed by me and a few others